### DISCUSSION TITLE
posts/frp/

### DISCUSSION THREAD

**Discussion by giscus** [Shitao5/shitao-blog#47] [Comments]
# posts/frp/

有了新电脑后希望它也能像云服务器一样，在电脑开机并且连着家里 WiFi 的情况下能随时访问。于是想起了之前听说但没有实践过的内网穿透技术。
[&hellip;] 打个比方，内网穿透就是在内部网络（如家里的 WiFi）和外部网络（如移动网络）中间搭建一个桥梁，让外部网络中的设备可以连接内部网络中的设备。最常见的例子或许是智能家居：出门在外的时候可以用手机控制家里连接着 WiFi 的电器。 &hellip;

https://shitao5.org/posts/frp/

**Comment by CyrusYip** [comment]
也可以试试 Zerotier 和 Tailscale，我之前的家用服务器就用它们，不用额外的服务器也能用。

**Reply by Shitao5** [reply]
好滴谢谢叶寻！俺看看学习下它们不用额外服务器的原理。
